src/constructs/aws/Webhook.ts

```diff
@@ -28,12 +28,14 @@ const WEBHOOK_DEFINITION = {
         insecure: { type: "boolean" },
         path: { type: "string" },
         eventType: { type: "string" },
+        method: { type: "string" },
```

Comment:
I'd be a bit more precise here, in order to avoid HTTP verbs without body - API Gateway would not throw an error for a `GET` method, but integration with Eventbridge would crash as no body payload would be accessible : 
```suggestion
        method: { enum: ["POST", "PUT", "PATCH"] },
```

Reply:
My latest changes have switched over to that!